can not export report

I'm gettingt the error. and can't export the html report file.
I have alos tried run the command ```npm insatll fs lodash```  previously
and how can I run  this command ```--reporter-html-export``` ? 

```
2018-02-17T12:29:37.4028268Z ##[section]Starting: Newman - Postman
2018-02-17T12:29:37.4034128Z ==============================================================================
2018-02-17T12:29:37.4034363Z Task         : Newman the cli Companion for Postman
2018-02-17T12:29:37.4034614Z Description  : Using Newman, one can effortlessly run and test a Postman Collections directly from the command-line. Now in a task!
2018-02-17T12:29:37.4034827Z Version      : 3.0.0
2018-02-17T12:29:37.4034990Z Author       : Carlo Wahlstedt
2018-02-17T12:29:37.4035213Z Help         : [More Information](https://github.com/carlowahlstedt/NewmanPostman_VSTS_Task/blob/master/README.md)
2018-02-17T12:29:37.4035430Z ==============================================================================
2018-02-17T12:29:38.0457685Z [command]C:\Windows\system32\cmd.exe /D /S /C "C:\npm\prefix\newman.cmd run --ssl-client-cert D:\a\1\s --ssl-client-key D:\a\1\s --reporter-html-template D:\a\1\s -r html -n 1 -e D:\a\1\s\environment.postman_environment.json D:\a\1\s\collection.postman_collection.json"
2018-02-17T12:29:45.7205887Z { Error: EISDIR: illegal operation on a directory, read
2018-02-17T12:29:45.7246710Z     at Object.fs.readSync (fs.js:675:18)
2018-02-17T12:29:45.7261430Z     at tryReadSync (fs.js:540:20)
2018-02-17T12:29:45.7325624Z     at Object.fs.readFileSync (fs.js:583:19)
2018-02-17T12:29:45.7330158Z     at new PostmanHTMLReporter (C:\npm\prefix\node_modules\newman\lib\reporters\html\index.js:53:42)
2018-02-17T12:29:45.7345469Z     at C:\npm\prefix\node_modules\newman\lib\run\index.js:319:68
2018-02-17T12:29:45.7372680Z     at arrayEach (C:\npm\prefix\node_modules\newman\node_modules\lodash\lodash.js:537:11)
2018-02-17T12:29:45.7385318Z     at Function.forEach (C:\npm\prefix\node_modules\newman\node_modules\lodash\lodash.js:9344:14)
2018-02-17T12:29:45.7386037Z     at C:\npm\prefix\node_modules\newman\lib\run\index.js:290:39
2018-02-17T12:29:45.7399318Z     at C:\npm\prefix\node_modules\newman\node_modules\postman-runtime\lib\backpack\index.js:56:34
2018-02-17T12:29:45.7411154Z     at C:\npm\prefix\node_modules\newman\node_modules\postman-runtime\lib\runner\index.js:96:20
2018-02-17T12:29:45.7412132Z     at extractRunnableItems (C:\npm\prefix\node_modules\newman\node_modules\postman-runtime\lib\runner\extract-runnable-items.js:135:35)
2018-02-17T12:29:45.7424512Z     at PostmanCollectionRunner.run (C:\npm\prefix\node_modules\newman\node_modules\postman-runtime\lib\runner\index.js:87:9)
2018-02-17T12:29:45.7452089Z     at C:\npm\prefix\node_modules\newman\lib\run\index.js:110:16
2018-02-17T12:29:45.7465570Z     at C:\npm\prefix\node_modules\newman\lib\run\options.js:246:9
2018-02-17T12:29:45.7466536Z     at C:\npm\prefix\node_modules\newman\node_modules\async\dist\async.js:473:16
2018-02-17T12:29:45.7467535Z     at C:\npm\prefix\node_modules\newman\node_modules\async\dist\async.js:3674:9 errno: -4068, code: 'EISDIR', syscall: 'read' }
2018-02-17T12:29:45.7558178Z newman warning: "html" reporter could not be loaded.
2018-02-17T12:30:26.8089948Z ##[error]C:\npm\prefix\newman.cmd failed with return code: 1
2018-02-17T12:30:26.8113958Z ##[section]Finishing: Newman - Postman
```

--- a/NewmanPostman/newmantask.ts
+++ b/NewmanPostman/newmantask.ts
@@ -14,11 +14,11 @@ function GetToolRunner() {
     newman.argIf(sslStrict, ['--insecure']);
 
     let reporterHtmlTemplate = tl.getPathInput('reporterHtmlTemplate', false, true);
-    newman.argIf(typeof reporterHtmlTemplate != 'undefined' && reporterHtmlTemplate, ['--reporter-html-template', reporterHtmlTemplate]);
-    let reporterHtmlExport = tl.getInput('reporterHtmlExport');
-    newman.argIf(typeof reporterHtmlExport != 'undefined' && reporterHtmlExport, ['--reporter-html-export', reporterHtmlExport]);
-    let reporterJsonExport = tl.getInput('reporterJsonExport');
-    newman.argIf(typeof reporterJsonExport != 'undefined' && reporterJsonExport, ['--reporter-json-export', reporterJsonExport]);
+    newman.argIf(typeof reporterHtmlTemplate != 'undefined' && tl.filePathSupplied('reporterHtmlTemplate'), ['--reporter-html-template', reporterHtmlTemplate]);
+    let reporterHtmlExport = tl.getPathInput('reporterHtmlExport');
+    newman.argIf(typeof reporterHtmlExport != 'undefined' && tl.filePathSupplied('reporterHtmlExport'), ['--reporter-html-export', reporterHtmlExport]);
+    let reporterJsonExport = tl.getPathInput('reporterJsonExport');
+    newman.argIf(typeof reporterJsonExport != 'undefined' && tl.filePathSupplied('reporterJsonExport'), ['--reporter-json-export', reporterJsonExport]);
     let reporterJUnitExport = tl.getPathInput('reporterJUnitExport', false, true);
     newman.argIf(typeof reporterJUnitExport != 'undefined' && tl.filePathSupplied('reporterJUnitExport'), ['--reporter-junit-export', reporterJUnitExport]);
     let reporters = tl.getInput('reporters');

--- a/NewmanPostman/task.json
+++ b/NewmanPostman/task.json
@@ -126,6 +126,27 @@
             "groupName": "requestOptions"
         },
         {
+            "name": "bail",
+            "type": "boolean",
+            "label": "Stop on first error",
+            "required": false,
+            "helpMarkDown": "Specify wether or not to stop a collection on first error",
+            "groupName": "requestOptions"
+        },
+        {
+            "name": "bailmodifier",
+            "type": "picklist",
+            "label": "Option",
+            "required": false,
+            "visibleRule": "bail=true",
+            "helpMarkDown": "'folder' will stop collection execution in case of incorrect folder configuration, 'failure' will gracefully stop collection on first test failure",
+            "groupName": "requestOptions",
+            "options": {
+                "folder": "folder",
+                "failre": "failure"
+            }
+        },
+        {
             "name": "sslClientCert",
             "type": "filePath",
             "label": "SSL Client Cert",

--- a/NewmanPostman/task.json
+++ b/NewmanPostman/task.json
@@ -182,7 +182,7 @@
         },
         {
             "name": "reporterHtmlExport",
-            "type": "string",
+            "type": "filePath",
             "label": "Reporter Html Export",
             "defaultValue": "",
             "required": false,
@@ -191,7 +191,7 @@
         },
         {
             "name": "reporterJsonExport",
-            "type": "string",
+            "type": "filePath",
             "label": "Reporter Json Export",
             "defaultValue": "",
             "required": false,

